lib/airbrake/sidekiq/retryable_jobs_filter.rb

```diff
@@ -0,0 +1,44 @@
+module Airbrake
+  module Sidekiq
+    ##
+    # Filter that can ignore notices from jobs that failed but will be retried
+    # by Sidekiq
+    class RetryableJobsFilter
+      if ::Sidekiq::VERSION < '5'
+        require 'sidekiq/middleware/server/retry_jobs'
+        DEFAULT_MAX_RETRY_ATTEMPTS = \
```

Comment:
I think this should be enough to support older Sidekiq versions (the constant was added somewhere around v2 but  moved places on v5) 🤔